apps/web/public/static/locales/en/common.json

```diff
@@ -1335,5 +1335,6 @@
   "limit_future_bookings":"Limit future bookings",
   "limit_future_bookings_description":"Limit how far in the future this event can be booked",
   "no_event_types": "No event types setup",
-  "no_event_types_description": "{{name}} has not setup any event types for you to book."
+  "no_event_types_description": "{{name}} has not setup any event types for you to book.",
+  "number_sms_notifications": "Phone number (SMS notifications)"
```

Comment:
Is it hacky to propose adding a non breaking space (`\u00a0`) between the words SMS and notifications? This will make them "one word" and add the line break after "number". That visually just looks a bit nicer 😁 

![CleanShot 2022-10-27 at 09 07 04@2x](https://user-images.githubusercontent.com/2969573/198214722-97495f40-1bf9-4deb-b5e6-255137809365.jpg)

Reply:
Wouldn't it be better if we structure it better by moving SMS notifications to a separate tag. That would also allow it to be styled differently(may be a little less bold and lighter color?)

Reply:
Good idea. Let's ask @Jaibles their opinion, since adding additional blocks might also make this look more "chaotic".